packages/dashmate/src/listr/tasks/setup/regular/registerMasternode/registerMasternodeWithCoreWallet.js

```diff
@@ -209,6 +212,18 @@ function registerMasternodeWithCoreWalletFactory(createIpAndPortsForm) {
 
       state = await task.prompt(prompts);
 
+      const portStatus = await providers.mnowatch.checkPortStatus(state.ipAndPorts.coreP2PPort);
+
+      if (portStatus !== PortStatusEnum.OPEN) {
```

Comment:
I believe we should do it as part of form validation and not accept invalid port

Reply:
The port is itself valid, it is just an additional check and reminder for masternode operators